### PR TITLE
Jupyter

### PR DESCRIPTION
**Pull request by ocefpaf**
The jupyter metapackage.

@minrk should I add you as a maintainer here?

### CODE REVIEW

**Comment by minrk** [comment]
> @minrk should I add you as a maintainer here?

yes, please.

File: recipes/jupyter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+  fn: jupyter-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/j/jupyter/jupyter-{{ version }}.tar.gz
+  sha256: d9dc4b3318f310e34c82951ea5d6683f67bed7def4b259fafbfe4f1beb1d8e5f
+
+build:
+  number: 0
+  script: python setup.py install
```
Comment (minrk) [inline recipes/jupyter/meta.yaml]: Can this be `pip install --no-deps .` so that appropriate metadata files are written?

Comment (ocefpaf) [inline recipes/jupyter/meta.yaml]: Testing that...

**Comment by ocefpaf** [comment]
> > @minrk should I add you as a maintainer here?

> yes, please.

Done!

File: recipes/jupyter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0.0" %}
```
Comment (jakirkham) [inline recipes/jupyter/meta.yaml]: Should this version have any meaning w.r.t the versions of the dependencies?

**Comment by jakirkham** [comment]
LGTM. Maybe could use a rebase.

**Pull request merged**